api/api.py

```diff
@@ -119,11 +119,58 @@ def get_route_stops_with_connections(route_id):
         route_stops.append(s)
     return route_stops
 
+def get_stop(stop_id):
+    result = db.engine.execute(
+        text(
+            """
+            SELECT 
+                osm_type,
+                osm_id,
+                name,
+                local_ref,
+                has_shelter,
+                has_bench,
+                has_tactile_paving,
+                has_departures_board,
+                is_wheelchair_ok,
+                st_asGeoJSON(ST_Transform(geom, 4326)) as geojson, 
+                routes_at_stop
+            FROM d_stops
+            WHERE osm_id = :stop_id
+        """
+        ),
+        stop_id=stop_id,
+    )
+    stops = []
+    for row in result:
+        d = OrderedDict(row)
+        s = OrderedDict([])
+        for k in d:
+            s[k] = d[k]
+        stops.append(s)
+    return stops
 
 class Index(Resource):
     def get(self):
         return {"links": [{"href": "api.py/route/<route_id>"}]}
 
+osm_types = {
+    0 : 'node',
+    1 : 'way',
+    2 : 'relation'
+}
+
+class Stop(Resource):
+    def get(self, stop_id):
+        if stop_id:
+            stops = get_stop(stop_id)
+            if len(stops) > 0 : 
```

Comment:
Space at the end of line
```suggestion
            if len(stops) > 0 :
```